pkg/cvo/sync_worker.go

```diff
@@ -803,6 +823,11 @@ func (w *SyncWorker) apply(ctx context.Context, work *SyncWork, maxWorkers int,
 
 	// update each object
 	errs := payload.RunGraph(ctx, graph, maxWorkers, func(ctx context.Context, tasks []*payload.Task) error {
+
+		capabilities := configv1.ClusterVersionCapabilitiesStatus{
+			EnabledCapabilities: capability.GetCapabilities(work.Capabilities.EnabledCapabilities),
+			KnownCapabilities:   capability.GetKnownCapabilities()}
```

Comment:
Can we replace this with:

```go
capabilties := capability.GetCapabilitiesStatus(work.Capabilities)
```

Then we could drop `GetCapabilities`.

Reply:
also, I'm not all that concerned about the weight of the `GetCapabilitiesStatus` call, but we could shift it up outside of `RunGraph`, because all the nodes in the graph will want the result, and none of them need to mutate `capabilities`.